packages/backend/src/core/PerpetualValidiumSyncService.ts

```diff
@@ -19,6 +20,7 @@ export class PerpetualValidiumSyncService implements IDataSyncService {
     private readonly perpetualCairoOutputCollector: PerpetualCairoOutputCollector,
     private readonly perpetualValidiumUpdater: PerpetualValidiumUpdater,
     private readonly withdrawalAllowedCollector: WithdrawalAllowedCollector,
+    private readonly transactionDownloader: TransactionDownloader | undefined,
```

Comment:
I suggest using "?" instead of "| undefined"

Reply:
Well, I would say that `| undefined` is a better pattern here. This way we enforce on someone to specify it, making sure that someone will not forget to pass it.